src/zenml/stack_deployments/azure_stack_deployment.py

```diff
@@ -17,15 +17,15 @@
 from typing import ClassVar, Dict, List
 
 from zenml.enums import StackDeploymentProvider
+from zenml.models import StackDeploymentConfig
 from zenml.stack_deployments.stack_deployment import ZenMLCloudStackDeployment
 
 
-# TODO: this class just implements the regions list, and is not suitable for other
-# deployment tasks.
 class AZUREZenMLCloudStackDeployment(ZenMLCloudStackDeployment):
     """Azure ZenML Cloud Stack Deployment."""
 
     provider: ClassVar[StackDeploymentProvider] = StackDeploymentProvider.AZURE
+    deployment: ClassVar[str] = "terraform"
```

Comment:
why do we make my life so much harder here? Now I have `cloud-formation`, `deployment-manager` and now `terraform` .. why cant we just say  `aws`, `gcp`, and `azure`... its really mind-boggling?

Reply:
renamed to `azure-cloud-shell` as agreed